clio-server/src/main/scala/org/broadinstitute/clio/server/service/GvcfService.scala

```diff
@@ -62,20 +62,17 @@ class GvcfService(
 
 object GvcfService {
```

Comment:
It looks like we still have duplication across the different service objects, it would be nice if we could eliminate this duplication too, at some point.

Reply:
That's @ApChagi's ticket